src/SwedbankPay/Core/Library/Methods/Mobilepay.php

```diff
@@ -23,6 +23,42 @@
  */
 trait Mobilepay
 {
+	/**
+	 * Check Mobilepay API Credentials.
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function checkMobilepayApiCredentials()
```

Comment:
The only thing that is different in these methods is the URI. Can you move the implementation to the SDK instead? The error handling should be the same across the board.

Reply:
Please move this to the SDK.